ci/terraform/oidc/ecc-signing-key.tf

```diff
@@ -14,7 +14,7 @@ resource "aws_kms_alias" "storage_token_signing_key_alias" {
   target_key_id = aws_kms_key.storage_token_signing_key_ecc.key_id
 }
 
-data "aws_iam_policy_document" "storage_token_signing_key_access_policy_with_orch_access" {
+data "aws_iam_policy_document" "storage_token_signing_key_access_policy" {
```

Comment:
After I made this rename I wasn't sure I liked it - I guess it depends where these KMS keys are going to end up - if they stay in auth, at some point we're going to want to remove cross account access

Reply:
This one can move to Orch accounts. We can check with Bravo, but we could probably just do it now given it's not in use yet

Reply:
Sorry I meant in general changing these names to remove `with_orch_access`

Reply:
I think , given auth's migration plans, it's unlikely any of these keys stay in this aws account long term. They'll either be moved to ours or theirs. I'm happy to just drop the `with_orch_access` and these will be cleaned up in time

Reply:
Nice, thanks